### ISSUE TITLE
Direct credit/debit card payment method

### ISSUE THREAD

**Issue by FeraruSilviuMarian** [top-gg/donatebot-issues#2]
#### Suggestion

We need a way to allow donors to be able to pay directly with their credit/debit card for people who doesn't have access to paypal due to their region.
The money could go to paypal or directly into the bank account through IBAN, I don't know how it works exactly but we need this.

**Comment by crashzk** [comment]
Exactly, some countries do not accept PayPal. It would be interesting to implement more forms of payments, there are several payment platforms that could be integrated into the project.
